packages/contracts-bedrock/scripts/checks/check-interfaces.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```

Reply:
This will probably catch the silent failure in CI 
<img width="1401" alt="image" src="https://github.com/user-attachments/assets/584296e2-9358-4c44-943e-e6621bda2e1d">

Reply:
Removed `jd`